terraform/grafana/provisioning/dashboards/containerSSH.json

```diff
@@ -445,7 +445,7 @@
           },
           "editorMode": "code",
           "exemplar": false,
-          "expr": "(node_memory_MemTotal_bytes{instance=\"gateway-vm:9100\"} - (node_memory_MemFree_bytes{instance=\"gateway-vm:9100\"} + node_memory_Cached_bytes{instance=\"gateway-vm:9100\"} + node_memory_Buffers_bytes{instance=\"gateway-vm:9100\"})) / node_memory_MemTotal_bytes{instance=\"gateway-vm:9100\"} * 100",
+          "expr": "(node_memory_MemTotal_bytes{instance=\"gateway-vm:9100\"} - node_memory_MemAvailable_bytes{instance=\"gateway-vm:9100\"}) / node_memory_MemTotal_bytes{instance=\"gateway-vm:9100\"} * 100",
```

Comment:
Replaced with `MemAvailable`.
https://stackoverflow.com/a/67302066/11554998
Maybe `MemAvailable` is more accurate?

Reply:
aha, so `MemAvailable` is an estimation how much memory can be used without swapping. yes let's use `MemAvailable`. :)